routes/users.ts

```diff
@@ -0,0 +1,12 @@
+import express from "express";
+import { join, getUser, editUser } from "../controller/usersController";
+
+const router = express.Router();
+
+router.use(express.json());
+
+router.get("/", getUser);
+router.post("/join", join);
+router.put("/", editUser);
```

Comment:
우선적으로는 같은 endpoint끼리, 그리고 HTTP Method별로 구분지어주면 좋을 것 같아요!
```suggestion
router.get("/", getUser);
router.put("/", editUser);
router.post("/join", join);
```

Reply:
아래처럼 다른 endpoint끼리는 개행을 넣어줘도 좋을 것 같구요!
```suggestion
router.get("/", getUser);
router.put("/", editUser);

router.post("/join", join);
```